README.md

```diff
@@ -14,34 +14,28 @@ Language: English | [日本語](./README.ja.md)
 Kubernetes History Inspector (KHI) is a rich log visualization tool for Kubernetes clusters. KHI transforms vast quantities of logs into an interactive, comprehensive timeline view.
 This makes it an invaluable tool for troubleshooting complex issues that span multiple components within your Kubernetes clusters.
 
-Any agents are not needed on your cluster. KHI provides following visualizations for troubleshooting just by loading logs on Kubernetes.
-
```

Comment:
`Agentless` is one of key points easily misunderstood in KHI. It doesn't need to be kept in the sentences, but we would like to emphasize KHI doesn't need to install anything on the cluster.